### PR TITLE
chore: set up workspace & init 'js-ui-core' package

### PR DESCRIPTION
**Pull request by movinsilva**
## Purpose
- Setup nx workspace
- Init '@asgardeo/js-ui-core' package

## Related Issue
- [SDK for Writing Custom Login Pages](https://github.com/wso2-enterprise/iam-engineering/issues/999)

### CODE REVIEW

**Review by dasuni-30 — APPROVED** [review]

File: packages/core/.prettierignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+/dist
+/node_modules
```
Comment (brionmario) [inline packages/core/.prettierignore:2]: Add new lines

File: packages/core/package.json [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  "main": "dist/cjs/index.js",
+  "module": "dist/esm/index.js",
+  "umd": "dist/umd/index.js",
+  "types": "dist/index.d.ts",
+  "author": "WSO2",
+  "license": "Apache-2.0",
+  "files": [
+    "dist"
+  ],
+  "homepage": "https://github.com/movinsilva/web-ui-sdks/tree/main/packages/core#readme",
```
Comment (brionmario) [inline packages/core/package.json:14]: Update the repo paths.

Comment (movinsilva) [reply packages/core/package.json]: ack, updated

**Review by dasuni-30 — APPROVED** [review]

File: packages/core/.prettierignore (hunk 1 above)
Comment (movinsilva) [reply packages/core/.prettierignore]: added with dbc2838e1d5bf91fc4306447bb56fb51fd014165

**Pull request merged**